en/docs/learn/basic-client-profile-with-playground.md

```diff
@@ -6,422 +6,98 @@ sample.
-                application. This action revokes all tokens issued for this
-                application. In order to activate the application, you have
-                to regenerate the consumer secret.
-
-            -   **Regenerate Secret:** Click to regenerate the secret key of
-                the OAuth application.
-
-            -   **Delete:** Click to delete the OAuth/OpenID Connect
-                Configurations.
-
-        !!! tip
-            The OAuth client key and client secret are stored in plain text. To
-            encrypt the client secret, access token and refresh token, do the
-            following:
-        
-            Open the `              deployment.toml             ` file found in the
-            `              <IS_HOME>/repository/conf             `
-            directory and change the following configuraiton
-        
-            ``` toml
-            [oauth]
-            token_storage_method="encrypted"
-            ```
+This application is used to request access tokens using the four OAuth2
+grant types:
 
-        After updating the configuration, make sure to restart the server
-        for the changes to be applied on WSO2 IS.
+-   [Authorization Code
+    Grant](../../learn/try-authorization-code-grant)
+-   [Client Credentials
+    Grant](../../learn/try-client-credentials-grant)
+-   [Implicit
+    Grant](../../learn/try-implicit-grant)
+-   [Try Password/Resource Owner
+    Grant](../../learn/try-password-grant)
 
 
-    !!! info "Related Topics"
-        See [Configuring OpenID Connect Single
-        Logout](../../learn/configuring-openid-connect-single-logout)
-        to configure single logout or session management with OpenID Connect.
+[OAuth 2.0](https://oauth.net/2/) has three main phases. They are;
+requesting an Authorization Grant, exchanging the Authorization Grant
+for an Access Token and accessing the resources using this Access Token.
+[OpenID Connect](http://openid.net/connect/) is another identity layer
+on top of OAuth 2.0. OAuth applications can get authentication event
+information over the IDToken and can get the extra claims of the
+authenticated user from the OpenID Connect UserInfo endpoint.  
 
-        See [Delegated Access
-        Control](../../learn/oauth-concepts)
-        for more information on working with OAuth2/OpenIDConnect. See the
-        following topics for samples of configuring delegated access control:
+!!! tip
+    For more information on OAuth2-OpenID Connect
+    refer, [Configuring OAuth2-OpenID Connect Single-Sign-On](../../learn/configuring-oauth2-openid-connect-single-sign-on)
 
-        -   [OAuth 2.0 with WSO2
-            Playground](../../learn/setting-up-the-sample-webapp)
-        -   [Setting up a SAML2 Bearer Assertion Profile for OAuth
-            2.0](../../learn/setting-up-a-saml2-bearer-assertion-profile-for-oauth-2.0)
+    After updating the configuration, make sure to restart the server
+    for the changes to be applied on WSO2 IS.
```

Comment:
Do we need to restart identity server, since no configuration change is done? shall we remove this line

Reply:
Let's remove Line105-106, step 2, as it is inconsistent 

```
 Log in as the user you created and go to the
        [dashboard](../../learn/using-the-end-user-dashboard).
```

Reply:
there are configuration changes in setting up the playground sample

Reply:
But no need to restart the IS server, right? Restarting tomcat would be applicable.